stan/math/prim/err/check_unit_vector.hpp

```diff
@@ -48,6 +52,26 @@ void check_unit_vector(const char* function, const char* name,
   }
 }
 
+/**
+ * Check if the each element in a standard vector is a unit vector.
```

Comment:
Should say "unit Euclidean length" to specify what unit vector means here.  "unit_vector" is a type in the Stan language.